src/main/java/uk/ac/ebi/eva/pipeline/EvaPipelineJobLauncher.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2015-2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.batch.core.JobExecutionException;
+import org.springframework.batch.core.converter.DefaultJobParametersConverter;
+import org.springframework.batch.core.converter.JobParametersConverter;
+import org.springframework.batch.core.explore.JobExplorer;
+import org.springframework.batch.core.launch.JobLauncher;
+import org.springframework.batch.core.repository.JobRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.batch.JobLauncherCommandLineRunner;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+import uk.ac.ebi.eva.pipeline.runner.ManageJobsUtils;
+import uk.ac.ebi.eva.pipeline.runner.exceptions.NoJobToExecute;
+import uk.ac.ebi.eva.pipeline.runner.exceptions.NoParametersHaveBeenPassed;
+import uk.ac.ebi.eva.pipeline.runner.exceptions.NoPreviousJobExecution;
+import uk.ac.ebi.eva.pipeline.runner.exceptions.UnexpectedErrorReadingFile;
+import uk.ac.ebi.eva.pipeline.runner.exceptions.UnexpectedFileCodification;
+import uk.ac.ebi.eva.pipeline.runner.exceptions.UnrecognizedElement;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Properties;
+
+@Component
+public class EvaPipelineJobLauncher extends JobLauncherCommandLineRunner {
+
+    private static final Logger logger = LoggerFactory.getLogger(EvaPipelineJobLauncher.class);
+
+    public static final String SPRING_BATCH_JOB_NAME_PROPERTY = "spring.batch.job.names";
+    public static final String PROPERTY_FILE_PROPERTY = "properties.file";
+    private static final String PROPERTY_FORCE_NAME = "force.restart";
```

Comment:
This still refers to `properties` (now as `properties.path`). The main idea was to replace `properties` with `parameters` because that is the obvious term for the user, the fact that it is a properties file in secondary.